code/modules/organs/internal/_internal.dm

```diff
@@ -121,6 +121,7 @@
 	if(istype(owner) && (toxin_type in owner.chem_effects))
 		take_damage(owner.chem_effects[toxin_type] * 0.1 * PROCESS_ACCURACY, prob(1))
 	handle_regeneration()
+	tick_surge_damage() //Yes, this has to be here, since thi
```

Comment:
You have half a comment there. Probably meant to copy the same as in the other file?

> Yes, this being here is intentional since this proc does not call ..() unless the owner is null.

Reply:
how the fuck did this happen